packages/storybook/stories/BreakoutRooms.stories.mdx

```diff
@@ -0,0 +1,135 @@
+import { Meta, Source } from '@storybook/addon-docs';
```

Comment:
Remove this, we have (as of 10 minutes ago) fully moved to storybook 8!

Reply:
Sounds good. Removed.